iea15mw/power_curve/envs/eagle_setup_env.sh

```diff
@@ -0,0 +1,7 @@
+source /nopt/nrel/ecom/hpacf/env.sh # if you aren't already doing this
+module use /projects/hfm/psakiev/spack-manager/modules
+module load exawind-master/2024-02-20
```

Comment:
Should we make a note about this is subject to change?

Reply:
I don't intend to change it. Ideally we should keep these up to date along the same lines ad the comment on the mesh.